linera-views/src/graphql.rs

```diff
@@ -23,7 +33,13 @@ impl<K: async_graphql::OutputType, V: async_graphql::OutputType> async_graphql::
     for Entry<K, V>
 {
     fn type_name() -> Cow<'static, str> {
-        format!("Entry_{}_{}", K::type_name(), V::type_name()).into()
+        format!(
+            "Entry_{}_{}_{:08x}",
+            K::type_name(),
```

Comment:
Do you mean to replace `[]` and `!` in another PR?

Reply:
I didn't, I just forgot :sweat_smile: I've added a commit, thanks for the catch!